### ISSUE TITLE
[Accessibility] Terminal help menu does not reflect the recent keybindings update for next/previous commands

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#189234]
Type: <b>Bug</b>

On Mac, in terminal, press Option+F1. It says:

- Go to Next Command (Command+DownArrow)
- Go to Previous Command (Command+UpArrow)

This has been recently updated as: Option+Up/DownArrows.

VS Code version: Code - Insiders 1.81.0-insider (Universal) (9800cf6dd6bf4634889d60720ef46a400f3a7298, 2023-07-28T12:09:45.183Z)
OS version: Darwin arm64 22.5.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|3, 4, 8|
|Memory (System)|16.00GB (0.11GB free)|
|Process Argv|--crash-reporter-id 1ab95d0d-f097-4265-add0-6d817fc5ec03|
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (90)</summary>

Extension|Author (truncated)|Version
---|---|---
android-dev-ext|ade|1.3.2
Bookmarks|ale|13.4.1
openscad|Ant|1.2.1
spellright|ban|3.0.116
zoterolatex|bna|0.4.1
mermaid-markdown-syntax-highlighting|bpr|1.5.2
doxdocgen|csc|1.4.0
dscodegpt|Dan|2.1.15
vscode-markdownlint|Dav|0.51.0
vscode-eslint|dba|2.4.2
vscode-quick-select|dba|0.2.9
vscode-deno|den|3.19.1
gitlens|eam|14.1.1
EditorConfig|Edi|0.16.4
prettier-vscode|esb|9.19.0
vscode-google-translate|fun|1.4.13
codespaces|Git|1.14.13
copilot|Git|1.99.294
copilot-chat|Git|0.5.2023072701
remotehub|Git|0.60.0
vscode-github-actions|git|0.25.8
vscode-pull-request-github|Git|0.68.1
easy-snippet|inu|0.6.3
path-autocomplete|ion|1.24.1
latex-workshop|Jam|9.13.3
lilypond-syntax|jea|0.1.1
scheme|jea|0.2.0
better-cpp-syntax|jef|1.17.2
google-search|kam|0.0.1
vscode-lua-format|Koi|1.3.8
lilypond-formatter|lhl|0.2.3
lilypond-pdf-preview|lhl|0.2.8
lilypond-snippets|lhl|0.1.1
vslilypond|lhl|1.7.3
zotero|mbl|0.1.10
git-graph|mhu|1.30.0
vscode-docker|ms-|1.26.0
black-formatter|ms-|2023.4.1
flake8|ms-|2023.6.0
isort|ms-|2023.11.12061012
python|ms-|2023.12.0
vscode-pylance|ms-|2023.7.40
jupyter|ms-|2023.6.1101941928
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
remote-containers|ms-|0.305.0
remote-ssh|ms-|0.102.0
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.80.2
vscode-remote-extensionpack|ms-|0.24.0
azure-repos|ms-|0.36.0
cmake-tools|ms-|1.14.34
cpptools|ms-|1.16.3
cpptools-extension-pack|ms-|1.3.0
js-debug-nightly|ms-|2023.7.2417
remote-repositories|ms-|0.38.1
vscode-github-issue-notebooks|ms-|0.0.129
vscode-selfhost-test-provider|ms-|0.3.15
vscode-serial-monitor|ms-|0.10.0
vsliveshare|ms-|1.0.5877
resourcemonitor|mut|1.0.7
autodocstring|njp|0.6.1
pandocciter|not|0.10.3
shiny-python|Pos|0.1.2
shinyuieditor|pos|0.4.3
quarto|qua|1.90.0
r-debugger|RDe|0.5.4
java|red|1.21.0
vscode-xml|red|0.26.1
r|REd|2.8.1
multi-command|ryu|1.6.0
vscode-deepl|soe|1.0.6
abc-music|sof|0.4.0
lua|sum|3.6.26
latex-utilities|tec|0.4.10
chatgpt|tim|1.1.2
cmake|twx|0.0.17
errorlens|use|3.12.0
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-arduino|vsc|0.6.0
vscode-java-debug|vsc|0.52.0
vscode-java-dependency|vsc|0.23.0
vscode-java-pack|vsc|0.25.12
vscode-java-test|vsc|0.39.0
vscode-maven|vsc|0.41.0
markdown-all-in-one|yzh|3.5.1
grammarly|znc|0.22.1

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythontb:30258533
vsc_aacf:30263846
pythonptprofiler:30281269
vsdfh931cf:30280410
vshan820:30294714
vscod805:30301674
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30404738
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30618038
showlangstatbar:30737417
24365598:30687740
pythonfmttext:30716741
pythoncmvfstr:30726892
fixshowwlkth:30771523
showindicator:30766888
pythongtdpath:30726887
i26e3531:30792625
gsofa:30797620
welcomedialog:30800244
pythonnosmt12:30779711
pythonidxpt:30768918
pythonnoceb:30776497
copilotsettingc:30767685
e537b577:30772214
h0f32768:30792099
synctok:30799130
dsvsc013:30777762
dsvsc014:30777825
cmakesidepanelv2:30779593
diffeditorv2:30786206

```

</details>

**Comment by meganrogge** [comment]
Yes @roblourens reported this, but I cannot find that issue, so we'll track it here. This is because of this line

https://github.com/microsoft/vscode/blob/8e333c3084522fcff7af9cd060dacaf80b04ccfc/src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibilityHelp.ts#L59-L61
